File: curso-python/ex018.py
from math import radians,sin,cos,tan

#leia um angulo qualquer
angulo = float(input("Valor do ângulo em graus: "))

angulo_radianos = radians(angulo)

seno = sin(angulo_radianos)
cosseno = cos(angulo_radianos)
tangente = tan(angulo_radianos)

print(f"Seno: {seno:.2f}")
print(f"Cosseno: {cosseno:.2f}")
print(f"Tangente: {tangente:.2f}")

File: curso-python/javascript-exercicios/ex018.js
const prompt = require("prompt-sync")();

// Lê o valor do ângulo em graus
const angulo = parseFloat(prompt("Valor do ângulo em graus: "));

// Converte para radianos
const anguloRadianos = angulo * Math.PI / 180;

// Calcula seno, cosseno e tangente
const seno = Math.sin(anguloRadianos);
const cosseno = Math.cos(anguloRadianos);
const tangente = Math.tan(anguloRadianos);

// Exibe os resultados com 2 casas decimais
console.log(`Seno: ${seno.toFixed(2)}`);
console.log(`Cosseno: ${cosseno.toFixed(2)}`);
console.log(`Tangente: ${tangente.toFixed(2)}`);
